components/mixins/opportunity/opportunities-details.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/opportunity-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "opportunity Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/opportunity"],
+  "description": "XDM business opportunity details.",
+  "definitions": {
+    "opportunity-details": {
+      "properties": {}
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/opportunity-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-opportunity"
```

Comment:
Why a B2B-Opportunity datatype and not just having those fields be defined in this mixin? When is the datatype to be used outside of this mixin?